styleguide/components/sidebar/sidebar-docs.mdx

```diff
@@ -0,0 +1,40 @@
+import { Story, Preview } from '@storybook/addon-docs';
+
+# Sidebar
+
+<Preview>
+  <Story id="components-sidebar--example" />
+</Preview>
+
+## With Additional Content
+
+<Preview>
+  <Story id="components-sidebar--with-additional-content" />
+</Preview>
+
+## Using with Rails
+
+If you are using the `citizens_advice_components` gem, you can call the component from within a template using:
+
+```rb
+= render CitizensAdviceComponents::Sidebar.new(title: "Related Content") do |c|
+    c.section_title(title: "Applying to the EU settlement scheme", url: "/immigration#h-applying-to-the-eu-settlement-scheme")
+    c.sidebar_links([
```

Comment:
It might make the docs a bit easier to read if the links array was a bit shorter and the link text inside was also a bit shorter / more illustrative (eg `{ title: "Link 1", url: "/link-1" }` or smth,

Reply:
Good point. Should I also change the links array in other places it's used (spec, preview and example files) or is it okey to only have it illustrative in the docs?

Reply:
It's okay for it to be illustrative in the docs, I don't think you need to change the others